### ISSUE TITLE
Visualization: RegionCoder.dictionary of Geochart doesn't cover IR-32 Alborz province, Iran?

### ISSUE THREAD

**Issue by alikarimi64** [google/google-visualization-issues#2806]
Google charts is a very useful plugin that i have been using for a very long time, came across this issue where RegionCoder.dictionary of Geochart doesn't cover IR-32 Alborz province, Iran.

https://www.gstatic.com/charts/regioncoder/0/geocodes/ir.js

**Comment by Saeid2016** [comment]
> 
> 
> Google charts is a very useful plugin that i have been using for a very long time, came across this issue where RegionCoder.dictionary of Geochart doesn't cover IR-32 Alborz province, Iran.
> 
> https://www.gstatic.com/charts/regioncoder/0/geocodes/ir.js

Yes this is a bug in Google charts plugin. Please fix it.

**Comment by dlaliberte** [comment]
There is an update of the regioncoder data available now.  See [posting about the regioncoderVersion option here](https://groups.google.com/d/msg/google-visualization-api/aN2CgZ1Qxno/sY_LrdcUBQAJ). 

I don't know if this new version of the regioncoder data will work better for this particular question.  Please try it and post either way.  Thanks.

**Comment by msr930** [comment]
> There is an update of the regioncoder data available now. See [posting about the regioncoderVersion option here](https://groups.google.com/d/msg/google-visualization-api/aN2CgZ1Qxno/sY_LrdcUBQAJ).
> 
> I don't know if this new version of the regioncoder data will work better for this particular question. Please try it and post either way. Thanks.

No, the new version doesn't cover IR-32 Alborz province, Iran either.
